apps/wallet-connect/src/assets/safe-app-connect.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="44" height="44" viewBox="0 0 44 44" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
https://components.gnosis-safe.io/?path=/docs/data-display-icon--icons

There is already `apps` here

Reply:
yep, that's right. Will use that